src/pipecat/transports/services/daily.py

```diff
@@ -1034,32 +1005,26 @@ async def capture_participant_audio(
         self,
         participant_id: str,
         audio_source: str = "microphone",
+        sample_rate: int = 16000,
     ):
-        await self._client.capture_participant_audio(
-            participant_id, self._on_participant_audio_data, audio_source
-        )
+        if self._streaming_started:
+            await self._client.capture_participant_audio(
+                participant_id, self._on_participant_audio_data, audio_source, sample_rate
+            )
+        else:
+            self._capture_participant_audio.append((participant_id, audio_source, sample_rate))
```

Comment:
Shouldn't we also call `await self._client.capture_participant_audio` here? Or is there a reason it's not needed?

I'm still missing something about how this would work if a third participant joins.

Reply:
If "stream on start" is not enabled, we don't want to capture the audio, so we just stop the list of participants that keep joining until `start_streaming()` is called.

Reply:
My question is more like this: if `audio_in_stream_on_start` is `enabled` and we invoke `start_audio_in_streaming`, we begin capturing audio from the participants currently in the meeting.

But if another participant joins later and we invoke `await self._client.capture_participant_audio`, will their audio also be captured ?

Reply:
That's what `if self._streaming_started` does. If you have already called `start_audio_in_streaming()` it will set `_streaming_started` to True, so you capture right away.